nene_core:
  ros__parameters:
    cmd_steering_topic: "cmd_steering"
    cmd_speed_topic: "cmd_speed"
    max_cmd_speed: 0.6
    min_cmd_speed: 0.4
    turn_right_cmd_steering: -0.7
    turn_left_cmd_steering: 0.7
    gain: 1.00

steering_predictor:
  ros__parameters:
    image_topic: "realsense_camera/image_uncompressed"

lane_detector:
  ros__parameters:
    image_topic: "realsense_camera/image_uncompressed"

test_video_publisher:
  ros__parameters:
    video_filepath: "/home/parallels/dev_ws/src/lane_keeping_assist/share/test_videos/light_reflection.mp4"
